extensions/jira-search/src/types.ts

```diff
@@ -0,0 +1,2 @@
+export const issueFilters = <const>["allIssues", "issuesInOpenSprints", "myIssues", "myIssuesInOpenSprints"]
+export type IssueFilter = (typeof issueFilters)[number]
```

Comment:
One more thing: Haven't seen before, that you're not using the array `issueFilters` anywhere, so I don't see a reason to define it at all. Declaring the type would be enough:

```suggestion
export type IssueFilter = "allIssues" | "issuesInOpenSprints" | "myIssues" | "myIssuesInOpenSprints"
```